### PR TITLE
mysql -> postgresql, add get nearby houses endpoint

### PR DESCRIPTION
**Pull request by Tura-lab**

### CODE REVIEW

**Review by Daniel-W1 — CHANGES_REQUESTED** [review]

File: src/interfaces/housePost.interface.ts [hunk 1]
```diff
@@ -1,12 +1,11 @@
...
 export interface HousePost {
   id?: number;
   hostId: number;
-  hostName: string;
-  hostProfileMedia: string;
   title?: string;
   description: string;
   street_address: string;
-  absolute_location: string;
+  absolute_location?: string; // eg. 'Point (3 4)'
```
[the remark this answers is not in the document]
Comment (Tura-lab) [reply src/interfaces/housePost.interface.ts:8]: > why did you make this optional, i think we need it in all houses
I thought it made sense to make it optional.

[the remark this answers is not in the document]
Comment (Tura-lab) [reply src/interfaces/housePost.interface.ts:8]: > and its still string like "(lat, long)" ?

prisma doesn't support the sql POINT type, so I kept it as string, but we can parse this string format into sql POINT type in our sql queries.

[the remark this answers is not in the document]
Comment (Daniel-W1) [reply src/interfaces/housePost.interface.ts:8]: we get the absolute address from the street address the users add, we infer it internally from google places API, it's not like users have to do sth.

File: src/dtos/housePost.dto.ts [hunk 2]
```diff
@@ -1,18 +1,11 @@
+import { HouseType } from '@prisma/client';
 import { IsString, IsOptional, IsNotEmpty, IsNumber, IsBoolean, IsArray } from 'class-validator';
 
 export class CreateHousePostDto {
   @IsNumber()
   @IsNotEmpty()
   public hostId: number;
 
-  @IsString()
```
[the remark this answers is not in the document]
Comment (Tura-lab) [reply src/dtos/housePost.dto.ts:8]: > why did you remove this ?

You mean those fields from the host? why would we need them? we can just reference with the hostId and join

File: src/interfaces/housePost.interface.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (Tura-lab) [reply src/interfaces/housePost.interface.ts:8]: I see I see. updating

**Pull request merged**